doc/cprover-manual/contracts-history-variables.md

```diff
@@ -0,0 +1,36 @@
+# History Variables
+
+### Syntax
+
+```c
+__CPROVER_old(*identifier*)
+```
+
+Refers to the value of a given object in the pre-state of a function within the
+_ensures_ clause.
+
+**Important.** This predicate may be used only within the context of `__CPROVER_ensures`.
+
```

Comment:
`__CPROVER_old` is not a predicate -- it's a function.

Reply:
Fixed it.